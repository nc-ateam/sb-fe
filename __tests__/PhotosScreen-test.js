import React from "react";
import "react-native";
import PhotosScreen from "../screens/PhotosScreen";

import renderer from "react-test-renderer";

describe("<PhotosScreen />", () => {
  it("renders without crashing", () => {
    const rendered = renderer.create(<PhotosScreen />).toJSON();
    expect(rendered).toBeTruthy();
  });

  it("creates a screenshot", () => {
    const rendered = renderer.create(<PhotosScreen />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
