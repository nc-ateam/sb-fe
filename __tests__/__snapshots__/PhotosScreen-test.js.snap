// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PhotosScreen /> creates a screenshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#000",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
        "padding": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
        }
      }
    >
      Camera permissions not granted - cannot open camera preview.
    </Text>
  </View>
</View>
`;
